src/test/java/seedu/storemando/logic/parser/ReminderCommandParserTest.java

```diff
@@ -25,12 +25,6 @@ public void parse_invalidSingleArg_throwsParseException() {
             String.format(MESSAGE_INVALID_COMMAND_FORMAT, ReminderCommand.MESSAGE_USAGE));
     }
 
-    @Test
```

Comment:
Just wondering, does removing this test decrease the test coverage? Would it be better if you change it to assertParseSuccess since we are allowing negative numbers now.

Reply:
Will add more test :)